fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import UserList from './user-list';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then(response => response.json())
+      .then(data => {
+        this.setState({ users: data });
+        this.setState({ isLoading: !this.state.isLoading });
```

Comment:
You can update multiple values in state with a single `setState` call. Update this to use one `setState` call